packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1155,11 +1156,17 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
   @override
   bool get selectAllEnabled => widget.toolbarOptions.selectAll;
 
+  // Inform the widget that the value of clipboardStatus has changed.
```

Comment:
nit: move this comment also inside the set state to explain why the empty set state is OK here.

Reply:
Missed that, thanks.